site/styles/_nav.scss

```diff
@@ -11,8 +10,16 @@
   top: 0;
   width: 100%;
   z-index: 9;
+  background: linear-gradient(rgba($top-gradient-color,1) 0%, rgba($top-gradient-color,0) ($navbar-brand-padding-y * 2) + 20px);
```

Comment:
I don't like this relying on `$navbar-brand-padding-y`... I think that variable may get removed when we add self-service branding.  Why not use a percentage?

Reply:
@PeculiarGoat This gradient goes down from the top of the `<header>` tag which I have made 100% the height of the screen when you open the menu. If the gradient had a percentage of 100%, then it would extend accross the screen. I chose not to hard code a value for the gradient because it wouldn't cover the nav properly if the logo is large.

Reply:
@PeculiarGoat I've fixed the issue by adding the gradient to `navbar-brand-container-tablet` instead of `<header>` when at the tablet breakpoints.